intel-retail/automated-self-checkout
# Add capi yolov8 ensemble pipeline profile

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**499**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- build capi-yolov8-ensemble pipeline profile: `make build-capi_yolov8_ensemble`
- start camera-simulator: `make run-camera-simulator`
- run pipeline: `PIPELINE_PROFILE="capi_yolov8_ensemble" DEVICE="CPU" RENDER_MODE=1 sudo -E ./run.sh --platform core inputsrc rtsp://127.0.0.1:8554/camera_0`
- run unit tests: ` make unit-test-profile-launcher` should see ALL PASSED
- run smoke tests should see ALL PASSED: ` make run-smoke-tests`

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
